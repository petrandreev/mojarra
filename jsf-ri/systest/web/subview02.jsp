<!--
 Copyright 2004 Sun Microsystems, Inc. All rights reserved.
 SUN PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
-->
<%@ page contentType="text/html" language="java" %>
<%@ taglib prefix="c" uri="http://java.sun.com/jstl/core" %>
<%@ taglib prefix="f" uri="http://java.sun.com/jsf/core" %>
<%@ taglib prefix="h" uri="http://java.sun.com/jsf/html" %>

<f:view>
<html>
<head>
<title>subview02</title>
</head>
<body>
<p>
<h:outputText value="Begin test <c:import> with subview tag in importing page"/>
</p>

<p><f:subview id="foo02">
<c:import url="foo02.jsp"/>
</f:subview></p>


<p><h:outputText value="subview02"/></p>

<p><f:subview id="bar02">
<c:import url="bar02.jsp"/>
</f:subview>
</p>

<p><h:outputText value="End test <c:import> with subview tag in importing page"/></p>
</body>
</html>
</f:view>
