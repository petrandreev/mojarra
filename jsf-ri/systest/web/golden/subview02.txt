<!--
 Copyright 2004 Sun Microsystems, Inc. All rights reserved.
 SUN PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
-->






<html>
<head>
<title>subview02</title>
</head>
<body>
Begin test &lt;c:import&gt; with subview tag in importing page

foo02<!--
 Copyright 2004 Sun Microsystems, Inc. All rights reserved.
 SUN PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
-->










subview02

bar02<!--
 Copyright 2004 Sun Microsystems, Inc. All rights reserved.
 SUN PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
-->










End test &lt;c:import&gt; with subview tag in importing page
</body>
</html>

